build-farm/platform-specific-configurations/linux.sh

```diff
@@ -78,7 +78,18 @@ fi
 
 if [ "${ARCHITECTURE}" == "arm" ]
 then
-  export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --with-jobs=4 --with-memory-size=2000"
+  if lscpu | grep aarch64; then
+     echo Validating 32-bit environment on 64-bit host - perhaps it is a docker container ...
+     if ! file /bin/ls | grep "32-bit.*ARM"; then
+       echo /bin/ls is not a 32-bit system. This configuration is invalid without extra work
```

Comment:
Maybe: `is not configured as a 32-bit system`?

Reply:
Hmmm. it's not a system configuration option really, although reading it back I don't like "32-bit system" either since it's explicitly checking `/bin/ls` only. Perhaps `/bin/ls is not 32-bit - 32-bit userland tools are needed to be able to build natively for ARCHITECTURE=arm`? Happy for other suggestions though

Reply:
Updated